install_config/install/docker_registry.adoc

```diff
@@ -1199,20 +1199,20 @@ Settings] in the
 link:../../install_config/persistent_storage/persistent_storage_nfs.html[Persistent
 Storage Using NFS] topic for details.
 
-*Pull of internally managed image fails with `not found` error*
+*Pull of Internally Managed Image Fails with `not found` Error*
 
-This will happen if the pulled image is pushed to an image stream different
-from the one it's being pulled from. It's a result of
+If the pulled image is pushed to an image stream different from the one it is
+being pulled from, then this error will occur. It is a result of the
 link:../../architecture/core_concepts/builds_and_image_streams.html#adding-tag[re-tagging]
-of a built image into arbitrary image stream like this:
+of a built image into an arbitrary image stream like this:
```

Comment:
The sentence structure of the first sentence isn't the best, and the 'it is' in the second is expletive. I'd suggest:

"This error occurs when the pulled image is pushed to an image stream different from the one it is being pulled from. This is caused by re-tagging a built image into an arbitrary image stream:"